python-main.js

```diff
@@ -618,6 +816,19 @@ function web_editor(config) {
                 $(".helpsupport_container").css("display", "none");
             }
         });
+
+        $("#command-flash").click(function () {
+            doFlash();
+        });
+        $("#command-serial").click(function () {
+            doSerial();
+        });
+        $("#request-repl").click(function () {
+            daplink.serialWrite("\x03"); 
+        });
+        $("#command-help").click(function () {
+            $(".helpsupport_container").toggle();
+        });
```

Comment:
Adding a note here for future rebases and merges:
I assume when this handler was added the original one (a few lines above this) was removed. Some commit resolution ends up adding this handler without removing the other one, so when the help button is clicked the first one will open the help info and this second one will immediately close it.

Reply:
Ended up pushing commit b422f0feff26b8561db53a17dda778e0041a2b41 to this branch, for the beta release. But it is worth keeping in mind this change in case it causes conflicts in the future.